chia/data_layer/data_layer_wallet.py

```diff
@@ -140,7 +140,7 @@ def id(self) -> uint32:
     async def create_new_dl_wallet(
         cls: Type[_T_DataLayerWallet],
         wallet_state_manager: WalletStateManager,
-        wallet: Wallet,
+        wall: Wallet,
```

Comment:
I would make the interface read well and the local variables do any needed gymnastics.

Reply:
right, this name is part of the function signature..